pages/api/proxy/[...path].js

```diff
@@ -2,15 +2,12 @@ import httpProxy from 'http-proxy'
 import Cookies from 'cookies'
 import url from 'url'
 
-// Get the actual API_URL as an environment variable. For real
-// applications, you might want to get it from 'next/config' instead.
-const API_URL = process.env.API_URL
-
 const proxy = httpProxy.createProxyServer()
 
 export const config = {
 	api: {
 		bodyParser: false,
+		url: 'http://localhost:3000/api'
```

Comment:
This config object is for Next.js, we shouldn't pollute it with custom properties

Reply:
Ah, I couldn't find where it was used, so I thought it was just your config file 😆